### PR TITLE
Demo of a fake update PR

### PR DESCRIPTION
**Pull request by JLLeitschuh**

### CODE REVIEW

File: gradlew [hunk 1]
```diff
@@ -80,7 +80,7 @@ case "`uname`" in
     ;;
 esac
 
-CLASSPATH=$APP_HOME/gradle/wrapper/gradle-wrapper.jar
+CLASSPATH=$APP_HOME/gradle/wrapper/gradlе-wrapper.jar
```
Comment (eskatos) [inline gradlew:83]: ⚠️ 👀

File: gradlew.bat [hunk 2]
```diff
@@ -1,100 +1,100 @@
...
+
+:win9xME_args_slurp
+if "x%~1" == "x" goto execute
+
+set CMD_LINE_ARGS=%*
+
+:execute
+@rem Setup the command line
+
+set CLASSPATH=%APP_HOME%\gradle\wrapper\gradlе-wrapper.jar
```
Comment (eskatos) [inline gradlew.bat:82]: ⚠️ 👀 
This one is less apparent

**Review by eskatos — COMMENTED** [review]
> `  Expected to find at least 1 Gradle Wrapper JARs but got only 0`
https://github.com/JLLeitschuh/playframework/pull/1/checks?check_run_id=389599822#step:3:6

File: gradlew.bat (hunk 2 above)
Comment (JLLeitschuh) [reply gradlew.bat:82]: Agreed

**Review by Shane3221 — COMMENTED** [review]
uses: gradle/wrapper-validation-action@v1

name: "Validate Gradle Wrapper"
on: [push, pull_request]

jobs:
  validation:
    name: "Validation"
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: gradle/wrapper-validation-action@v1

**Review by QUONt3 — COMMENTED** [review]
[fake/JLL/fake-demo](https/github)

**Review by MIRELOSZKA196 — COMMENTED** [review]
F

**Pull request closed without merging**